examples/full-example/isomorphic/base/main.js

```diff
@@ -3,7 +3,6 @@
    See the accompanying LICENSE file for terms. */
 import App from './components/app';
 import React from 'react'; // eslint-disable-line
-import 'fake';
```

Comment:
hmm can you leave this? this is testing the "shim" concept. Yes, it doesn't work in Mendel v1.